.gitignore

```diff
@@ -25,6 +25,11 @@
 /dist
 .mplconfig
 
+# INSTALL.md recommends a venv that should not be committed
+venv
+.venv
+uv.lock
```

Comment:
Are we supposed to commit the lock file to keep versions the same for all developers?  I'm not that familiar with `uv`, so this is a genuine question.